Cargo.toml

```diff
@@ -21,7 +21,7 @@ r2d2-diesel = "1.0.0"
 taglib2-sys = { path = "taglib2-sys" }
 error-chain = "0.11.0"
 walkdir = "2.1.4"
-indicatif = "0.9.0"
+indicatif = { git = "https://github.com/mitsuhiko/indicatif.git", rev = "950091d1b1683a88e01c4d4975f591009f56322b" }
```

Comment:
Why did you change this?

Reply:
Answered my own question. The ability to print above the progress bar is not yet released:
https://github.com/mitsuhiko/indicatif/pull/41